charts/s3gw/values.yaml

```diff
@@ -41,7 +41,7 @@ imageRegistry: "ghcr.io/aquarist-labs"
 imagePullPolicy: "Always"
 
 imageName: "s3gw"
-imageTag: "latest"
+imageTag: "v0.1.0"
```

Comment:
What about `imageTag_ui`, should that stay `latest`?

Reply:
Chart version and images versions can diverge: suppose that you have to fix only the chart, you would increase the chart version keeping the current stable versions for the s3gw core and s3gw UI. 
Atm, the latest stable available image for s3gw core is `0.1.0`.
For the UI we still haven't an official image, so it can still be fine to leave `latest`.
Btw, if today we will release an image for UI, this must change.